ccmlib/scylla_cluster.py

```diff
@@ -133,9 +139,9 @@ def start_nodes(self, nodes=None, no_wait=True, verbose=False, wait_for_binary_p
     def start(self, no_wait=True, verbose=False, wait_for_binary_proto=None,
               wait_other_notice=None, jvm_args=None, profile_options=None,
               quiet_start=False):
-        args = locals()
```

Comment:
This is a change needed in CCM, regardless of our work,
without it, this part of code is failing under debugger